packages/orchestrator/internal/sandbox/checks.go

```diff
@@ -2,19 +2,36 @@ package sandbox
 
 import (
 	"context"
+	"encoding/json"
 	"fmt"
 	"io"
 	"net/http"
 	"time"
 
 	"github.com/e2b-dev/infra/packages/shared/pkg/consts"
 	"github.com/e2b-dev/infra/packages/shared/pkg/utils"
+	"golang.org/x/mod/semver"
+)
+
+const (
+	healthCheckInterval      = 10 * time.Second
+	metricsCheckInterval     = 5 * time.Second
+	minEnvdVersionForMetrcis = "0.1.3"
 )
 
 func (s *Sandbox) logHeathAndUsage(ctx *utils.LockableCancelableContext) {
+	healthTicker := time.NewTicker(healthCheckInterval)
+	metricsTicker := time.NewTicker(metricsCheckInterval)
+	defer func() {
+		healthTicker.Stop()
+		metricsTicker.Stop()
+	}()
+
+	go s.LogMetrics(ctx)
```

Comment:
please add comment explaining, this is to get metrics on start on the sandbox 🙏🏻